### PR TITLE
ipmitool: deprecate

### PR DESCRIPTION
**Pull request by SMillerDev**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
https://github.com/ipmitool/ipmitool is archived

### CODE REVIEW

File: Formula/ipmitool.rb [hunk 1]
```diff
@@ -16,6 +16,8 @@ class Ipmitool < Formula
     sha256 x86_64_linux:   "a0a993a436ef12c14707d60293d37233dd90e0e80909c461f262c19874951d32"
   end
 
+  deprecate! date: "2022-12-16", because: :repo_archived
```
Comment (ZhongRuoyu) [inline Formula/ipmitool.rb:19]: Why this date, though? The last commit was made on 2023-01-12; the archive date should be after that.

https://github.com/ipmitool shows "Updated on Feb 9", so maybe 2023-02-09?

Comment (SMillerDev) [reply Formula/ipmitool.rb:19]:
```suggestion
  deprecate! date: "2023-02-09", because: :repo_archived
```

**Comment by Bo98** [comment]
Bit of a weird one this since it's technically not deprecated upstream but rather the developers got banned from GitHub.

**Comment by SMillerDev** [comment]
The repo says "public archive" so not gonna bother with that kind of nitpicking.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**